crates/hotshot-state-prover/src/circuit/types.rs

```diff
@@ -0,0 +1,40 @@
+//! Types for verification circuit
```

Comment:
nit: personally, I prefer to just keep these into `circuit.rs` file. don't see much benefits of keeping it separate as we don't have lots of types anyway.

Reply:
Everything is merged in one file now.